common/web/types/src/ldml-keyboard/pattern-parser.ts

```diff
@@ -51,11 +51,10 @@ export class MarkerParser {
    * Marker sentinel as a string - U+FFFF
    */
   public static readonly SENTINEL = String.fromCodePoint(constants.marker_sentinel);
-
   /**
-   * Matches all markers.
+   * Marker code as a string - U+0008
    */
-  public static readonly SENTINEL_ALL_MARKERS = this.SENTINEL + this.SENTINEL;
+  public static readonly CODE_DEADKEY = String.fromCodePoint(constants.marker_code_deadkey);
```

Comment:
```suggestion
  public static readonly MARKER_CODE = String.fromCodePoint(constants.marker_code);
```

I reckon let's avoid polluting the ldml keyboard code with 'deadkey' refs.

Reply:
Hm… though, then it matches the rest of the codebase.  

But, I suppose a (1) jsdoc comment and (2) unit tests asserting that it's the same, could be enough documentation.

Reply:
Core already uses the term marker synonymously with deadkey. So ldml core can and should use marker. Let's keep the deadkey term internal to the kmx core.

Reply:
Fine with me. My only question is whether kmx_file.h should have an alias as well.

Reply:
I don't think it needs to at this point. kmx_file.h is used only by the KMX processor which only believes in deadkeys, and it's quite old so we don't want it getting confused ;-)